ocs_ci/ocs/resources/pod.py

```diff
@@ -3831,3 +3832,52 @@ def get_pod_used_memory_in_mebibytes(podname):
             if memory_value_with_unit.endswith("Mi"):
                 memory_value = int(memory_value_with_unit.replace("Mi", ""))
                 return memory_value
+
+
+def get_age_of_cluster_in_days():
+    """
+    Get age of the cluster in days.
+    1. Get creation time by executing oc cmd on cluster
+    2. Get current time from the ceph tools pod
+    3. Calculate time difference between two times
+    4. Convert the time into days
+
+    Returns:
+        time_diff_in_days: (int) returns number of days the cluster has been running
+
+    """
+
+    cmd = "get namespace kube-system -o jsonpath='{.metadata.creationTimestamp}'"
+    creation_time = OCP().exec_oc_cmd(command=cmd, out_yaml_format=False)
+
+    ct_pod = get_ceph_tools_pod()
+    cephcmd = 'date -u +"%Y-%m-%dT%H:%M:%SZ"'
+    current_time = ct_pod.exec_cmd_on_pod(command=cephcmd, out_yaml_format=False)
+
+    d1 = datetime.fromisoformat(creation_time[:-1])
+    d2 = datetime.fromisoformat(current_time.strip()[:-1])
+    time_difference_in_sec = (d2 - d1).total_seconds()
+    seconds_per_day = 24 * 60 * 60
+    time_diff_in_days = time_difference_in_sec / seconds_per_day
```

Comment:
d2-d1 won't workout for me here. Because if there are any extra hours or minutes or seconds in the days, I have to round figure it to an another day. (d2-d1). days will give days in integer where I am losing the extra hrs/mns/secs. I want it including seconds. So that math.ceil() will give an additional day for me even if it just crossed with seconds.
ex: d2-d1 --> datetime.timedelta(days=3, seconds=20185)
(d2-d1).days -->3
time_diff_in_days --> 3.233622685185185
math.ceil(time_diff_in_days) = 4